### PR TITLE
add claroty content

### PR DESCRIPTION
**Pull request by spsocprime**
## Proposed Changes
Added analytic content for Claroty data connector

### CODE REVIEW

**Comment by v-sabiraj** [comment]
@aprakash13 ,please review Analytic Rules, thanks.

**Review by shainw — CHANGES_REQUESTED** [review]
Queries look good, but I think there is an opportunity to simplify the number of distinct queries by using custom alert names and base them off of the Event types you are creating individual queries on.  Most of the queries with the ORs in them, can simply be made into a single query that looks for all of the different matches and labels the Alert name and Description based on the value that is triggered.  See - https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide#alertdetailsoverride

**Comment by vmanojreddy** [comment]
Workbook query is failing, please look into it.

image

**Comment by vmanojreddy** [comment]
@aprakash13 ,please review Analytic Rules, thanks.

File: Solutions/Claroty/Analytic Content/ClarotyNewAsset.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+id: 6c29b611-ce69-4016-bf99-eca639fee1f5
+name: Claroty - New Asset Alert
+description: |
+  'Triggers when new asset discovered.'
```
Comment (aprakash13) [inline Solutions/Claroty/Analytic Content/ClarotyNewAsset.yaml:4]: Can you please double check if the logic detects when critical deviation from baseline occurs or if it detects new asset discovered?

File: Solutions/Claroty/Hunting Queries/ClarotyUnapprovedAccess.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+id: de0fca32-85f3-45df-872e-41e980e5d8d3
+name: Claroty - Unapproved access
+description: |
+  'Query searches for unaprooved access events.'
```
Comment (aprakash13) [inline Solutions/Claroty/Hunting Queries/ClarotyUnapprovedAccess.yaml:4]: Please correct the typo in the spelling of unapproved.

**Review by aprakash13 — CHANGES_REQUESTED** [review]
A few suggestions in relation to this PR

The folder path for analytic rules should be Solutions/Claroty/Analytic **Rules**/ vs Solutions/Claroty/Analytic Content/. Can you please change the folder structure?

In the project statements can you please check if there are other relevant fields that we could output that will help the customer in their analysis.

For future, I would suggest that for detections that have just a single condition, such as | where EVentOriginalType =~ "Unapproved" that you move these all into a single custom alert that changes the alert name and description based on the content of the query - https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide#alertdetailsoverride

File: Solutions/Claroty/Analytic Content/ClarotyNewAsset.yaml (hunk 1 above)
Comment (spsocprime) [reply Solutions/Claroty/Analytic Content/ClarotyNewAsset.yaml]: fixed

File: Solutions/Claroty/Hunting Queries/ClarotyUnapprovedAccess.yaml (hunk 2 above)
Comment (spsocprime) [reply Solutions/Claroty/Hunting Queries/ClarotyUnapprovedAccess.yaml]: fixed

**Comment by spsocprime** [comment]
> https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide#alertdetailsoverride



> Workbook query is failing, please look into it.
> 
> image

Queries are failing because you do not have events in your environment and this field is extracted dynamically from event data.

**Comment by vmanojreddy** [comment]
Workbook looks good. Can merge once approved by @aprakash13.

**Comment by NikTripathi** [comment]
Merging as everything looks good.

**Pull request merged**